rest-api/dao/participant_summary_dao.py

```diff
@@ -379,6 +380,34 @@ def to_client_json(self, model):
 
     return result
 
+  def to_client_csv(self, model):
+    from flask import Response
+    import StringIO
+    # from unicode_csv import UnicodeWriter
+
+
+    def iter_csv(data):
+      line = StringIO.StringIO()
+      writer = csv.writer(line)
+      for csv_line in data:
```

Comment:
Each participant summary returned should be a single line of CSV. We shouldn't iterate through multiple lines for a single summary... which is what it sounds like this was going to do.